src/Execute/Device.ts

```diff
@@ -32,10 +33,20 @@ import {DeviceSpec} from '../Backend/Spec';
 class Device {
   name: string;
   spec: DeviceSpec;
+  availableExecutor: Set<Executor>;
```

Comment:
If there is no 'unavailable executor', why not naming it as 'executors'?

Reply:
> If there is no 'unavailable executor'

That's right. but I want to mark those `Executor' could run on this `Device`, so I name like that.

Reply:
um...I think keep this on would be nice. if `Executor` which not `satisfied()` Device's Spec means `unavailable executor`. so, I think available will be a minimum expression for those `executors`.